about.md

```diff
@@ -0,0 +1,78 @@
+---
+layout: default
+---
+
+## Why are we doing this?
+
+Remote Hack helps developers and technologists get together to create solutions to Remote working, collaboration, and socialising. To go full-meta, we'll be hacking on tools for Remote hacking.
+
+As social distancing sets in and many of us switch to full-time Remote, this event is a chance to hack on things that make Remote a little easier.
+
+## What isn't it?
+
+- **A competition.** There are no prizes, and no ceremonies. At the end of the day there will be the
+  opportunity to show what you've been hacking on, if you'd like to.
+- **A hackathon.** We'll be taking breaks, and exercising self-care.
+- **Aiming to fix the world.** Things are pretty dire for a lot of people right now, and If you'd like
+  to hack on something directly related to COVID-19 then 🙌, but you're equally welcome to build something fun, or
+  just learn something new.
+- **About delivering polished software.** This is a chance to collaboratively explore an idea, see if
+  something _could_ work, or learn something new.
+
+## What will we be hacking on?
+
+The theme of the day is "remote". Add your ideas as issues to our [GitHub issue tracker](https://github.com/remotehack/remotehack.github.io/issues) for inspiration on the day. We'll come together in the morning and split out into small groups based on what interests us.
+
+We'll meet in our Slack to say hi, and be ready to kick off on Zoom at 9:30am. We encourage you to take breaks, and we'll be pausing for lunch around 12:30 (we recommend burritos 🌯 to support your local Mexican takeaway). We'll come together again around 4:30pm to share our experiences and anything we've built.
+
+During the day we'll be on Zoom and Slack, and encourage you to drop in for a chat. We can also help you get set up in groups to collaborate remotely.
+
+## What do I need?
+
+As long as you have a machine to hack on and an internet connection, you're good to go. We'll be collaborating on Slack and Zoom, and there are a bunch of really good tools for ad-hoc screen-sharing and collaboration.
+
+We've started putting together a list of [remote hacking resources](https://github.com/remotehack/resources). Feel free to add anything you think might be useful!
+
+## Code of Conduct
+
+All conferences and events should be friendly, safe and inclusive spaces for anyone to attend, and Remote Hack is no different. All organisers and attendees must abide by the [Berlin Code of Conduct](https://berlincodeofconduct.org).
+
+## The Podcast
+
+We'll be recording interviews with the participants throughout the day. Follow along if you like (the episodes are super short...around 2 minutes each).
+
+NOTE: being interviewed is NOT necessary to participate in the hackday.
+
+come follow along at [the RSS feed](https://remotehack.space/live/feed), or just search for "Airquotes the Podcast Airquotes" in your podcatcher.
```

Comment:
while we're here, I had totally forgot that Spotify is basically the only place you can find the Podcast at the moment...could we update "your podcatcher" to "spotify"?

Reply:
Ah, totally.

Do you know, perhaps we should move the podcast to a page of its own?  It's a really nice way of finding out about the kind of stuff we're working on.

Reply:
Podcast on its own page totally works!

Reply:
# 🙌 

[![image](https://user-images.githubusercontent.com/51385/83550717-2efa6900-a4ff-11ea-8954-22880593102e.png)](https://deploy-preview-45--wizardly-wiles-9029ce.netlify.app/podcast)